tests/test_household.py

```diff
@@ -225,6 +228,54 @@ def test_get_tr(TR, j, p, method, expected):
     assert np.allclose(test_value, expected)
 
 
+# Set up test for get_rm
+(
+    RM1,
+    RM2,
+    RM3,
+    RM4,
+    expected_rm1,
+    expected_rm2,
+    expected_rm3,
+    expected_rm4,
+) = utils.safe_read_pickle(
+    os.path.join(CUR_PATH, "test_io_data", "RMrm_test_tuple.pkl")
```

Comment:
@jdebacker. The objects in this pickle file (`RMrm_test_tuple.pkl`) are big. I do not recommend that we put them in line in `test_household.py`. The pickled object is a tuple with 8 elements, the contents of which have the following shape:
- scalar
- scalar
- numpy array (400,)
- numpy array (400,)
- numpy array (80,)
- numpy array (80,)
- numpy array (400, 80)
- numpy array (400, 80)

Reply:
@rickecon But is there any reason one needs 400x80?  Wouldn't a 4x3 specification have the same effect of testing that functions work?  You can see such specifications being done in other tests.

Reply:
@jdebacker. Yes, you're right that a 4x3 test is doable and more manageable. But the case we really care about is the 400x80 case. I can't think of a reason why the 4x3 case shouldn't give us what we need to know. But it feels better to me to have a big test.

You tell me what you prefer. If you want me to restructure this test to be a 4x3, I'll do it. I think my preference is to keep the 520KB pickle file 400x80 version. But I defer to you on testing.

Reply:
Ok - let's merge this, but I may go back and simplify in a future PR.  I want to do a thorough review of where we need the pkl files and where we don't.  I want to avoid them as much as possible.